README.md

```diff
@@ -59,11 +80,13 @@ evervault = Evervault::Client.new(api_key: <YOUR-API-KEY>)
 result = evervault.encrypt_and_run(<CAGE-NAME>, { hello: 'World!' })
 ```
 
-## API Reference
+## Reference
+
+At present, there are seven methods available in the Ruby SDK.
```

Comment:
I don't think this gives us anything and we'll have to keep coming around and updating it any time we make a change.
```suggestion
```

Reply:
went with: "The Evervault Ruby SDK exposes seven methods."